definitions/push-images/push-images-pipeline.yaml

```diff
@@ -0,0 +1,28 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: push-images-pipeline
+spec:
```

Comment:
I think it would make more sense to have some params listed as Pipeline params so they can be passed to the pipeline itself (with default values using the values on lines 11, 15, 18, 21 if you want) and then use those to pass to the task. So the task param would be like
```
- name: srcImageURL
  value: "$(params.srcImageURL)"
```

Reply:
That makes sense. We'll keep that in mind when we re-implement the pipeline. For now this one was deleted.